Package.swift

```diff
@@ -90,8 +90,8 @@ let package = Package(
         ),
         .binaryTarget(
             name: "PayPalCheckout",
-            url: "https://github.com/paypal/paypalcheckout-ios/releases/download/0.110.0/PayPalCheckout.xcframework.zip",
-            checksum: "e9895c202b090a7bde5c47685e96aef68b6f334e3f3798a79dd49b32c81fe130"
+            url: "https://github.com/paypal/paypalcheckout-ios/releases/download/0.112.1/PayPalCheckout.xcframework.zip",
+            checksum: "b1da7584ccb2964e3bddab1871e7f4b4121b4639f7de91b3988371a7e2fce546"
```

Comment:
Im updating MXO to the latest version in PPCP. Whats the diference between the way its done here than what we are doing in PPCP? [Here is the diff](https://github.com/paypal/iOS-SDK/pull/142/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR38).

Reply:
Good question - i think adding it as a dependency would be preferable, but it may be treated as a dependency for the entire package (rather than a dependency for just one target). @scannillo may have more info.

Reply:
Yeah, so when we link it as a dependency (in PPCP) vs a binary target (in BT) we end up pulling it in for everything. So for example when we link the library as a dependency in BT, if a merchant was using just Card for example, it would still link NXO even though it wasn't being used by that module.

Likely, we want to take a similar approach on PPCP that we opted to take on BT and move to a binary target, but we can discuss this further as a team.